packages/readiness/bundle/config/values.yaml

```diff
@@ -5,3 +5,5 @@ deployment:
   hostNetwork: false
   nodeSelector: null
   tolerations: []
+  webhookServerPort: 9443
+  tlsCipherSuites: "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
```

Comment:
Please add description for the new values in `packages/readiness/README.md`.

Reply:
Done